src/esw/brushless.py

```diff
@@ -82,16 +82,19 @@ def __init__(self, can_id: int, transport):
         self._command = CommandData(
             position=CommandData.POSITION_FOR_VELOCITY_CONTROL,
             velocity=CommandData.ZERO_VELOCITY,
-            torque=CommandData.MAX_TORQUE,
+            torque=CommandData.DEFAULT_TORQUE,
```

Comment:
Can't you just let the default constructor do this? I think it may be worth commenting what setting command to this means